frontend/src/components/AnalysisView/AnalysisViewHeader.vue

```diff
@@ -40,21 +44,29 @@ export default {
       type: Array,
       required: true,
     },
-    mondayLink: {
-      type: String,
-      default: '',
-      required: false,
-    },
-    phenotipsLink: {
-      type: String,
-      default: '',
-      required: false,
+    third_party_links: {
```

Comment:
With the third-party links code streamlined now into a single array, this works better to be integrated directly into RosalutionHeader to avoid duplication between the AnalysisViewHeader and AnnotationViewHeader.  An example of this is the actions menu that is a prop with RosalutionHeader. 

Migrate this code into the RosalutionHeader to consolidate it and allow for future headers to utilize the icons also.